pipeline-scripts/buildlib.groovy

```diff
@@ -126,6 +126,11 @@ def setup_venv() {
 
 }
 
+setup_doozer() {
+    doozerWorking = "${env.WORKSPACE}/doozer_working"
+    env.DOOZER_WORKING_DIR = doozerWorking
```

Comment:
hopefully you tested this... does this env var actually get passed in to shell scripts? I thought you needed `withEnv` (https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#setting-environment-variables).

If the working dir isn't set, doozer will still function, at least until it fills up `/tmp` which won't take long... please make sure a working dir is set :)